jwql/edb/engineering_database.py

```diff
@@ -64,14 +64,13 @@
 
 from jwst.lib.engdb_tools import ENGDB_Service
 from jwql.utils.constants import MIRI_POS_RATIO_VALUES
+from jwql.utils.constants import ON_GITHUB_ACTIONS
 from jwql.utils.credentials import get_mast_base_url, get_mast_token
 from jwql.utils.utils import get_config
 
 MAST_EDB_MNEMONIC_SERVICE = 'Mast.JwstEdb.Mnemonics'
 MAST_EDB_DICTIONARY_SERVICE = 'Mast.JwstEdb.Dictionary'
 
-# Temporary until JWST operations: switch to test string for MAST request URL
-ON_GITHUB_ACTIONS = '/home/runner' in os.path.expanduser('~') or '/Users/runner' in os.path.expanduser('~')
 if not ON_GITHUB_ACTIONS:
```

Comment:
I'm surprised this hasn't been failing on RTD. Maybe RTD isn't looking here for some reason.

Reply:
Yeah, not sure. It looks as if it *should* be, but possibly there's been a change where ON_GITHUB_ACTIONS ends up resolving to True when the readthedocs build is running, at least in cases where there isn't a config.